src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -402,12 +402,13 @@
 					eventDate: ts.date
 				};
 
+				ts.url_params = {};
+
 				if ( ts.category ) {
 					ts.params['tribe_event_category'] = ts.category;
+					ts.url_params['tribe_events_cat'] = ts.category;
```

Comment:
Same thing here: `ts.params['tribe_events_cat']` to `ts.params.tribe_events_cat`